.github/workflows/release.yml

```diff
@@ -0,0 +1,37 @@
+on:
+  repository_dispatch:
+    types: [release]
+  schedule:
+    # We want the release to be at 9-10am Pacific Time
+    # We also want it to be 1 hour before the on-prem release
+    - cron:  '0 17 15 * *'
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    name: "Release a new version"
+    steps:
+        - id: calver
+          if: ${{ !github.event.client_payload.version }}
+          run: echo "::set-output name=version::$(date +'%y.%-m.0')"
+        - uses: actions/checkout@v2
+        - uses: getsentry/craft@master
+          if: ${{ !github.event.client_payload.skip_prepare }}
+          with:
+            action: prepare
+            version: ${{ github.event.client_payload.version || steps.calver.outputs.version }}
+          env:
+            DRY_RUN: ${{ github.event.client_payload.dry_run }}
+            GIT_COMMITTER_NAME: getsentry-bot
+            GIT_AUTHOR_NAME: getsentry-bot
+            EMAIL: bot@getsentry.com
+        - uses: getsentry/craft@master
+          with:
+            action: publish
+            version: ${{ github.event.client_payload.version || steps.calver.outputs.version }}
+          env:
```

Comment:
Do you need to pass the github token like in relay ?
https://github.com/getsentry/relay/blob/master/.github/workflows/release.yml#L36

Reply:
GitHub actions provide a token only effective on this repo and we use it by default. Relay touched some other repos so we needed a different token